src/index.spec.ts

```diff
@@ -28,17 +18,17 @@ describe('isEqual', function () {
     expect(isEqual(null, undefined)).to.equal(true);
     expect(isEqual(undefined, undefined)).to.equal(true);
     expect(isEqual(NaN, NaN)).to.equal(true);
-    expect(isEqual(null, NaN)).to.equal(true);
-    expect(isEqual(undefined, NaN)).to.equal(true);
+    expect(isEqual(null, NaN)).to.equal(false);
+    expect(isEqual(undefined, NaN)).to.equal(false);
   });
   it('should return "true" for objects with same properties', function () {
     expect(isEqual({ foo: 'foo', bar: true }, { bar: true, foo: 'foo' })).to.equal(true);
     expect(isEqual({ foo: null }, { foo: null })).to.equal(true);
     expect(isEqual({ foo: null }, { foo: undefined })).to.equal(true);
     expect(isEqual({ foo: undefined }, { foo: undefined })).to.equal(true);
     expect(isEqual({ foo: NaN }, { foo: NaN })).to.equal(true);
-    expect(isEqual({ foo: null }, { foo: NaN })).to.equal(true);
-    expect(isEqual({ foo: undefined }, { foo: NaN })).to.equal(true);
+    expect(isEqual({ foo: null }, { foo: NaN })).to.equal(false);
```

Comment:
Why was this changed?

If intentional, the test should be moved out of the test block that says it tests for `true`.